<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-45FQJ0I" failures="14" tests="42" name="Test" time="28.776" errors="0" timestamp="2022-09-12T13:57:20 CDT">
  <testcase classname="testSuite1.Module2" name="test001" time="0.005"/>
  <testcase classname="testSuite1.Module2" name="test002" time="0.001"/>
  <testcase classname="testSuite1.Module2" name="test003" time="0.001"/>
  <testcase classname="testSuite1.Module2" name="test004" time="0.001"/>
  <testcase classname="testSuite1.Module1" name="test001" time="0.001"/>
  <testcase classname="testSuite1.Module1" name="test002" time="0.0"/>
  <testcase classname="testSuite1.Module1" name="test003" time="0.004"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="ValidateEmpSal" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [5000.0] but found [5000.001]">
      <![CDATA[java.lang.AssertionError: expected [5000.0] but found [5000.001]
at testSuite2.EmployeeSalaryTest.ValidateEmpSal(EmployeeSalaryTest.java:90)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ValidateEmpSal -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpDiv" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [Devloper] but found [Devloper.]">
      <![CDATA[java.lang.AssertionError: expected [Devloper] but found [Devloper.]
at testSuite2.EmployeeSalaryTest.validateEmpDiv(EmployeeSalaryTest.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpDiv -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpDiv1" time="0.588">
    <failure type="java.lang.AssertionError" message="expected [Tester] but found [Tester.]">
      <![CDATA[java.lang.AssertionError: expected [Tester] but found [Tester.]
at testSuite2.EmployeeSalaryTest.validateEmpDiv1(EmployeeSalaryTest.java:176)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpDiv1 -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpDiv1" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [Dev] but found [Dev.]">
      <![CDATA[java.lang.AssertionError: expected [Dev] but found [Dev.]
at testSuite2.EmployeeSalaryTest.validateEmpDiv1(EmployeeSalaryTest.java:176)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpDiv1 -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpDiv1" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [Pwoner] but found [Pwoner.]">
      <![CDATA[java.lang.AssertionError: expected [Pwoner] but found [Pwoner.]
at testSuite2.EmployeeSalaryTest.validateEmpDiv1(EmployeeSalaryTest.java:176)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpDiv1 -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpDiv1" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [Dir] but found [Dir.]">
      <![CDATA[java.lang.AssertionError: expected [Dir] but found [Dir.]
at testSuite2.EmployeeSalaryTest.validateEmpDiv1(EmployeeSalaryTest.java:176)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpDiv1 -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpDiv1" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [VP] but found [VP.]">
      <![CDATA[java.lang.AssertionError: expected [VP] but found [VP.]
at testSuite2.EmployeeSalaryTest.validateEmpDiv1(EmployeeSalaryTest.java:176)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpDiv1 -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpId" time="0.0"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpId1" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.0"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.0"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.0"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.0"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.0"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpName1" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpNm" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpSalDP" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [500.0] but found [500.001]">
      <![CDATA[java.lang.AssertionError: expected [500.0] but found [500.001]
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpSalDP -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpSalDP" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [501.0] but found [501.001]">
      <![CDATA[java.lang.AssertionError: expected [501.0] but found [501.001]
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpSalDP -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpSalDP" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [502.0] but found [502.001]">
      <![CDATA[java.lang.AssertionError: expected [502.0] but found [502.001]
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpSalDP -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpSalDP" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [503.0] but found [503.001]">
      <![CDATA[java.lang.AssertionError: expected [503.0] but found [503.001]
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpSalDP -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpSalDP" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [504.0] but found [504.001]">
      <![CDATA[java.lang.AssertionError: expected [504.0] but found [504.001]
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpSalDP -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateEmpSalDP" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [505.0] but found [505.001]">
      <![CDATA[java.lang.AssertionError: expected [505.0] but found [505.001]
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateEmpSalDP -->
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateMaxInt" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateMinValue" time="0.001"/>
  <testcase classname="testSuite2.EmployeeSalaryTest" name="validateempDiv" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [ Product Owner] but found [ Product Owner.]">
      <![CDATA[java.lang.AssertionError: expected [ Product Owner] but found [ Product Owner.]
at testSuite2.EmployeeSalaryTest.validateempDiv(EmployeeSalaryTest.java:79)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validateempDiv -->
  <testcase classname="selenium.SeleniumBasicTest" name="validateSeleniumSetup" time="18.03"/>
  <testcase classname="selenium.Assignment_5" name="validateWebPageload" time="10.025"/>
</testsuite> <!-- Test -->
